### DISCUSSION TITLE
blog/zh/read-write-tools/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#9] [Announcements]
# blog/zh/read-write-tools/

Ladder@我的阅读写作工具 - GuangzhengLi

https://guangzhengli.com/blog/zh/read-write-tools/

**Comment by xj4920** [comment]
感谢详细描述了从搜集信息到主体比较阅读，最终到写作输出的过程。本人目前阅读书单与知识沉淀，简直一团浆糊，规划性差，执行力弱，逐渐被琐事占据，阅读时间甚少，思考甚浅。

**Comment by ghost** [comment]
我在网站上用了 raindrop 和 readwise 的API 来分享收藏的内容，感觉还挺方便的，没整合到一个页面。[Reading · Yuxi & Yuqi](https://yuxiyuqi.com/reading/)

**Reply by guangzhengli** [reply]
Cool～

**Comment by tonyljx** [comment]
写得太好了，点赞

**Comment by hansen20190912** [comment]
omnivore能将碎片化信息保存成全文么？现在貌似只能保存成摘要，我想保存成全文然后做笔记，做高亮然后保存到notion。有没有好办法呢？

**Comment by xcccer** [comment]
写作风格、格式都让人感觉很舒服。受益匪浅。

**Comment by markyun** [comment]
模版太好看了，我也用上啦

**Comment by liufeng** [comment]
有空时可以讲讲Cubox切换到Omnivore的故事吗？Omnivore我用着还没入门。开源的确不错，但浏览器插件似乎没法从网页上完成标注，所以我目前还是很犹豫。说实话Cubox一年的费用真不贵。

**Comment by Houlson** [comment]
谢谢分享，收获良多

**Comment by eatless** [comment]
好厉害，从知乎一直追过来，对于工作初期的我，非常受益。尝试跟随搭建一下自己的博客。
